SPEC.md

```diff
@@ -244,6 +245,7 @@ The following fields are declared the specification file, and all are required:
 * `maintainer` (`Object`) - data about the engine maintainer
   * `name` (`String`) - the name of the maintainer
   * `email` (`String`) - the email address of the maintainer
+* `languages` (`[String]`) - an array of programming languages that this engine is meant to analyze. Language agnostic engines can specify `"*"`
```

Comment:
What are the acceptable values for `languages`? Is it `Javascript`, `JS` or `JavaScript`?